cargo-progenitor/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "cargo-progenitor"
+version = "0.3.0"
+edition = "2021"
+license = "MPL-2.0"
+description = "An OpenAPI client generator"
+repository = "https://github.com/oxidecomputer/progenitor.git"
+readme = "../README.md"
+keywords = ["openapi", "openapiv3", "sdk", "generator", "proc_macro"]
+categories = ["api-bindings", "compilers"]
+build = "build.rs"
+
+[dependencies]
+progenitor = { version = "0.3.0", path = "../progenitor" }
+progenitor-client = { version = "0.3.0", path = "../progenitor-client" }
+progenitor-impl = { version = "0.3.0", path = "../progenitor-impl" }
+anyhow = "1.0"
+clap = { version = "4.2.7", features = ["derive"] }
+openapiv3 = "1.0.0"
```

weird this didn't update...
```suggestion
openapiv3 = "1.0.2"
```

---

I used copy/paste instead of `cargo add` :)